DevVehicle35-Motors/App/VanInteraction.cs

```diff
@@ -0,0 +1,134 @@
+﻿using DevVehicle35_Motors.Models;
+
+namespace DevVehicle35_Motors.App
+{
+    internal class VanInteraction
+    {
+        internal static int Vcapacity = 0;
+
+        public static void BuildVan()
+        {
+            string? opt;
+            int optN;
+            Van van = new Van();
+            Console.WriteLine(string.Empty);
+            Console.WriteLine("Hi! We will help you to find the Van of your dreams...");
+            BathroomChoise();
+            opt = Console.ReadLine();
+            optN = ValidateInput(opt, 1, 2, 1);
```

Comment:
Is not a good practice to introduce more than 2 parameters to the functions. 
I see they are constants, so you can declare them inside the function.